server/src/views/frame.js

```diff
@@ -222,6 +222,7 @@ class Frame extends React.Component {
 
   clickFullPageButton(e) {
     e.preventDefault();
+    history.pushState({state: "frame"}, "", "#frame");
```

Comment:
Yeah, that's what I realized.  But if we don't have any click handler, let the links point the user to `#frame` as they will by default, then put in a special case for `frame` in clientglue:refreshHash, I think that'd work well.